janusgraph-dist/janusgraph-dist-hadoop-2/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM openjdk:8
+
+ENV PATH "$JAVA_HOME/bin:$PATH"
+RUN apt-get update -y
+RUN apt-get install -y zip
+WORKDIR /var
+
+ARG server_zip
+ARG server_base
+ADD ${server_zip} /var
+RUN unzip -q ${server_base}.zip
+RUN ln -s ./${server_base} ./janusgraph
+WORKDIR /var/janusgraph
+EXPOSE 8182
+CMD ["./bin/gremlin-server.sh"]
```

Comment:
I suggest changing the command here to `./bin/gremlin-server.sh ./conf/gremlin-server/gremlin-server-berkeleyje-es.yaml` and creating the new YAML file.

The change to `gremlin-server.yaml` should be reverted so `./bin/janusgraph.sh start` still works as documented with Cassandra and ES.